common/src/api/internal/shared.rs

```diff
@@ -103,6 +103,17 @@ pub struct BgpPeerConfig {
     pub port: String,
     /// Address of the peer.
     pub addr: Ipv4Addr,
+    /// How long to keep a session alive without a keepalive in seconds.
+    /// Defaults to 6.
+    pub hold_time: Option<u64>,
```

Comment:
This is currently part of the data stored in the bootstore, but I think because these are all optional fields deserialization should still work. I would suggest testing the upgrade path though, where you initialize a rack with the old version and then install the new to see what happens.